pyproject.toml

```
@@ -12,9 +12,9 @@ dependencies = [
   "beautifulsoup4",
   "fire",
   "huggingface-hub",
+  "kokoro>=0.7.9",
   "llama-cpp-python",
   "loguru",
-  "outetts<0.3",
   "pydantic",
   "PyPDF2[crypto]",
   "python-docx",
```

We need to add `soundfile` as an explicit dep. It was being installed before as an indirect dep of outetts

```suggestion
  "python-docx",
  "soundfile",
```